delivery_transsmart/models/res_config.py

```diff
@@ -108,47 +108,87 @@ def get_transsmart_service(self):
     def get_transsmart_carrier_tag(self):
         return self.env['ir.model.data'].get_object('delivery_transsmart', 'res_partner_category_transsmart_carrier')        
```

Comment:
Not changed by you, but might better use ref() to get a record through ir.model.data.